Modules/_testcapi/immortal.c

```diff
@@ -0,0 +1,79 @@
+#include "parts.h"
+
+static PyObject *
+test_immortal_bool(PyObject *self, PyObject *Py_UNUSED(ignored))
+{
+    PyObject *objects[] = {Py_True, Py_False};
+    Py_ssize_t n = Py_ARRAY_LENGTH(objects);
+    for (Py_ssize_t i = 0; i < n; i++) {
+        PyObject* obj = objects[i];
+        assert(_Py_IsImmortal(obj));
+        Py_ssize_t old_count = Py_REFCNT(obj);
+        for (int j = 0; j < 10000; j++) {
+            Py_DECREF(obj);
+        }
+        Py_ssize_t current_count = Py_REFCNT(obj);
+        assert(old_count == current_count);
```

Comment:
You might want to separate this out as a helper function, since this part is repeated for all `test_*()` functions.

Reply:
Done!